docs/config/README.md

```diff
@@ -631,11 +631,12 @@ it would have been `nixpkgs/pkgs`.
 | ------------------------ | ----------------------------------------------- | -------------------------------------------------------------------------------- |
 | `truncation_length`      | `3`                                             | The number of parent folders that the current directory should be truncated to.  |
 | `truncate_to_repo`       | `true`                                          | Whether or not to truncate to the root of the git repo that you're currently in. |
-| `format`                 | `"[$path]($style)[$lock_symbol]($lock_style) "` | The format for the module.                                                       |
+| `format`                 | `"[$truncated]($style)[$path]($style)[$read_only]($read_only_style) "` | The format for the module.                                |
```

Comment:
Please document the new `$truncated` variable in the variables section.

Beyond that LGTM.

Reply:
Decided to remove the `$truncated` variable as it doesnt make sense to move it.  It's now just included in `$path` as it was before with the fish-style path truncation.